parser/src/cfg/sexpr.rs

```diff
@@ -181,18 +232,31 @@ impl<'a> Lexer<'a> {
     #[allow(clippy::new_ret_no_self)]
     /// `file_name` is used only for indicating a file, where
     /// a fragment of `source` that caused parsing error came from.
-    fn new(source: &'a str, file_name: &'a str) -> impl Iterator<Item = Spanned<TokenRes>> + 'a {
+    fn new(
+        source: &'a str,
+        file_name: &'a str,
+        ignore_whitespace_and_comments: bool,
+    ) -> impl Iterator<Item = Spanned<TokenRes>> + 'a {
         let mut lexer = Lexer {
             s: source,
             bytes: source.bytes(),
+            ignore_whitespace_and_comments,
+            line: 0,
+            last_newline_pos: 0,
         };
         let file_name: Rc<str> = Rc::from(file_name);
         let file_content: Rc<str> = Rc::from(source);
         iter::from_fn(move || {
             lexer.next_token().map(|(start, t)| {
+                let end = lexer.pos();
```

Comment:
Is this useful to keep around?

Reply:
Oops this is in the wrong spot, this should be for the commented out section below